### PR TITLE
Build ci actions

### PR DESCRIPTION
**Pull request by javidlulu**
### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing](../../paypayopa-sdk-php/blob/master/contributing.md) document?
* [ ] Have you read and signed the automated Contributor's License Agreement?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../paypayopa-sdk-php/pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by ashraf-km — APPROVED** [review]
approved

File: .github/workflows/build-ci.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Run Tests
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: make run_mock
+
+      - name: Before script
+        run: |
+          curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+          chmod +x ./cc-test-reporter
+          ./cc-test-reporter before-build
```
Comment (shreyanshp) [inline .github/workflows/build-ci.yml:29]: shouldn't this go before Run tests?

File: .github/workflows/build-ci.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+      - name: Upload coverage results to Coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: make coverall_upload
+
+      - name: Upload to CoPilot
+        run: bash <(curl -s https://copilot.blackducksoftware.com/ci/githubactions/scripts/upload)
```
Comment (shreyanshp) [inline .github/workflows/build-ci.yml:37]: I think you have missed codeclimate in post steps, please check the travis.yml once and make sure we have all the scripts and features migrated to github actions

**Review by shreyanshp — CHANGES_REQUESTED** [review]

File: .github/workflows/build-ci.yml (hunk 1 above)
Comment (javidlulu) [reply .github/workflows/build-ci.yml:24]: done

**Review by shreyanshp — APPROVED** [review]

**Pull request merged**